packages/client/src/views/SysAdmin/Config/Systems/Systems.tsx

```diff
@@ -663,7 +667,7 @@ export function SystemList() {
                               label: intl.formatMessage(
                                 integrationMessages.documentDetails
                               ),
-                              value: 'documents'
+                              value: 'supporting-documents'
```

Comment:
@euanmillar These 3 values need to be updated in `WebhookModal.tsx` as well, similar to this commit:https://github.com/opencrvs/opencrvs-core/pull/4138/commits/8084dbe0a613507a5964702ce7a00687680b3e18